server/src/db/migrations/1_initial-migrations.sql

```diff
@@ -1,6 +1,15 @@
-CREATE TABLE student (
-  student_id SERIAL PRIMARY KEY,
+CREATE EXTENSION "uuid-ossp";
+
+CREATE TABLE "user" (
+  user_id UUID PRIMARY KEY DEFAULT uuid_generate_v4(),
   first_name TEXT NOT NULL,
   last_name TEXT NOT NULL,
-  birth_year INT NOT NULL
+  role TEXT NOT NULL CHECK (role IN ('para', 'case_manager', 'admin'))
+);
+
+CREATE TABLE "student" (
```

Comment:
Let's also add an email field here, which can be used to uniquely identify students that share the same names